SU-HSDO/suhumsci
# Hsd8 497 revision cleanup

# READY FOR REVIEW

# Summary
- Cron task to delete old revisions but keep a configured number.

# Needed By (Date)
- 12/5

# Urgency
- low

# Steps to Test

1. checkout this branch
1. sync to a site `blt drupal:sync --site=[sitenmae]`
1. review a node to find one that has more than 5 revisions
1. run cron
1. review that node again, verify only 5 revisions exist

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
